test/e2e/networking.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Networking", func() {
 					{
 						Name:    contName,
 						Image:   "gcr.io/google_containers/busybox",
-						Command: []string{"ping", "-c", "3", "-w", "10", "google.com"},
+						Command: []string{"nc", "-w", "30", "google.com", "80"},
```

Comment:
I have had a bunch of problems with nc being flaky without a TTY.  Can you use curl or wget (I think busybox has wget but not curl)